pkg/controller/operators/olm/operatorgroup.go

```diff
@@ -160,6 +184,37 @@ func (a *Operator) syncOperatorGroups(obj interface{}) error {
 	return nil
 }
 
+func unique(logger *logrus.Entry, a, b []string) []string {
```

Comment:
Open Question: Anyone have a better name?

Reply:
[Set difference](https://en.wikipedia.org/wiki/Complement_(set_theory))?

Reply:
I support this setDifference.